### PR TITLE
IpPrefix supports ipv6, add unit test

### PR DESCRIPTION
**Pull request by qiluo-msft**

### CODE REVIEW

File: common/Makefile.am [hunk 1]
```diff
@@ -1,5 +1,6 @@
 INCLUDES = -I $(top_srcdir)
 
+bin_PROGRAMS = unittest
```
Comment (stcheng) [inline common/Makefile.am]: move to test/ folder.

File: common/ipaddress.h [hunk 2]
```diff
@@ -59,11 +59,6 @@ class IpAddress
                );
     }
 
-    inline bool operator!=(const IpAddress &o) const
```
Comment (stcheng) [inline common/ipaddress.h]: why remove not equal?

File: common/ipprefix.h [hunk 3]
```diff
@@ -24,7 +25,38 @@ class IpPrefix
...
-        return IpAddress(htonl((0xFFFFFFFFLL << (32 - m_mask)) & 0xFFFFFFFF));
+        switch (m_ip.getIp().family)
+        {
+            case AF_INET:
+            {
+                return IpAddress(htonl((0xFFFFFFFFLL << (32 - m_mask)) & 0xFFFFFFFF));
+            }
+            case AF_INET6:
+            {
+                ip_addr_t ipa;
```
Comment (stcheng) [inline common/ipprefix.h]: Use IpAddress class?

File: common/ipaddress.h (hunk 2 above)
Comment (qiluo-msft) [inline common/ipaddress.h]: Not necessary for interacting with standard libraries.

File: common/ipprefix.h (hunk 3 above)
Comment (qiluo-msft) [inline common/ipprefix.h]: It's more convenient to operate on underlying array directly.

**Comment by stcheng** [comment]
Thanks

**Pull request merged**